src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,1224 @@
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {add: { $add: [$int, $int2]}}}]
+            cursor: {}
+
+- Name: ExpressionArrayElemAt
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionArrayElemAt
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {arrayElemAt: { $arrayElemAt: [$array, ^RandomInt: {min: 0, max: 23}]}}}]
+            cursor: {}
+
+- Name: ExpressionFirst
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionFirst
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {first: { $first: $array}}}]
+            cursor: {}
+
+- Name: ExpressionBSONSize
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionBSONSize
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {bsonSize: { $bsonSize: $nestedObj}}}]
+            cursor: {}
+
+- Name: ExpressionCeil
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionCeil
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {ceil: { $ceil: $double}}}]
+            cursor: {}
+
+- Name: ExpressionCompare
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionCompare
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {cmp: { $cmp: ["$a", 0]}}}]
```

Comment:
Shouldn't this be referencing "$int" or something? "$a" doesn't appear to exist in the documents we are using here: https://github.com/mongodb/genny/pull/754/files#diff-89754a7ec9efc43b5c5760f4dab92a27778d285884d56f55710d09e71b1c5b37R40-R54

Reply:
Done.